Documentation/HowToGuides/Basics/AddingTheUnityXRCameraRig/README.md

```diff
@@ -39,13 +49,13 @@ Select the `UnityXRCameraRig` GameObject in the Unity3d Hierarchy window and cha
 
 ![Unity XR Configuration Tracking Space Type](assets/images/UnityXRConfigurationTrackingSpaceType.png)
 
-### Complete
+### Done
```

Comment:
```suggestion
## Done
```
Honestly I'm not sure which one we should do - on the one hand we describe what the output of the last step was, on the other hand it's independent from them but still comes after them. It doesn't feel like it should be part of `Let's Start`, so I think my suggested change is worth doing?

Reply:
Resolved by Slack discussions.